iaas-common/src/main/java/io/github/cloudiator/persistance/PricingPriceDimensionsModel.java

```diff
@@ -0,0 +1,84 @@
+package io.github.cloudiator.persistance;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import java.math.BigDecimal;
+
+@Entity
+public class PricingPriceDimensionsModel extends Model {
+    @Column(nullable = false, precision = 12, scale = 4)
+    private BigDecimal pricePerUnit;
+    @Column(nullable = false)
+    private String unit;
+    @Column(nullable = true)
```

Comment:
All this `@Column(nullable = true)` are redundant

Reply:
done